src/loaddb/load_db_value_converter.cpp

```diff
@@ -88,6 +88,11 @@ namespace cubload
 
     for (int i = 0; i < NUM_DB_TYPES; i++)
       {
+	for (int j = 0; j < NUM_LDR_TYPES; j++)
+	  {
+	    setters_[i][j] = NULL;
```

Comment:
I have two comments regarding this change:
  1. looks like a different issue.
  2. can we initialize as &mismatch directly? see get_conv_func

Reply:
my bad again, it is same issue, but second observation stands :)